src/components/calcite-panel/calcite-panel.scss

```diff
@@ -89,7 +89,7 @@
     justify-start
     sticky
     top-0
-    border-b-color-3
+    border-b-color-2
```

Comment:
Can you remove the duplicate full width style within this class?

(@julio8a I'd love to resurrect the border-side-style plugin I had added a while back but then was directed to remove. [It would](https://github.com/Esri/calcite-components/blob/49c0adfd3971b140e7de8ce1c8271513c4883825/tailwind.config.js#L263) allow us to style borders like `@apply border-l border-l-solid border-l-color-input;` in the near future.)

Reply:
@caripizza, let's discuss in our next core team meeting.